### PR TITLE
New skip-link toolkit

### PR DESCRIPTION
**Pull request by nasiamakryg**
This should help approaching skip links consistently across the company.
I avoided using absolute positioning with top, left values since this method creates problems for some screen readers.
In SN it looks like this: 
Screen Shot 2021-06-01 at 09 30 17

### CODE REVIEW

File: toolkits/global/packages/global-skip-link/HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# History
+
+## 1.0.0 (2021-05-28)
```
Comment (alexkilgour) [inline toolkits/global/packages/global-skip-link/HISTORY.md:3]: I would suggest a version of `0.1.0` to start with as this isn't yet implemented on nature

File: toolkits/global/packages/global-skip-link/scss/10-settings/_nature.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * @springernature/global-skip-link
+ * Default skin settings
+ *
+ */
```
Comment (alexkilgour) [inline toolkits/global/packages/global-skip-link/scss/10-settings/_nature.scss:5]: all brand configs should include `@default` first

**Comment by alexkilgour** [comment]
Have you tested that this works with the OneTrust overlay?

**Comment by nasiamakryg** [comment]
I faked on live nature.com, latest chrome and firefox and it was behaving as expected. Asked also @tulipdexter to link it and check locally since my project doesn't have yet any cookie consent banners. He didn't get back to me yet.

**Comment by alexkilgour** [comment]
> I faked it on live nature.com, latest chrome and firefox and it was behaving as expected. Asked also @tulipdexter to link it and check locally since my project doesn't have yet any cookie consent banners. He didn't get back to me yet.

That's cool. I think we can get this out there, then I can help try and get it tested (David has been super busy on another project btw, but I can help find someone to look at it)

**Review by alexkilgour — APPROVED** [review]

**Review by alexkilgour — APPROVED** [review]

**Pull request merged**